fcrepo-kernel-impl/src/test/java/org/fcrepo/integration/kernel/impl/FedoraResourceImplIT.java

```diff
@@ -452,6 +452,38 @@ public void testUpdateDenied() throws RepositoryException {
                         + " WHERE { }", new RdfStream());
     }
 
+    @Test (expected = IllegalArgumentException.class)
+    public void testInvalidSparqlUpdateValidation() throws RepositoryException {
+        final FedoraResource object =
+                containerService.findOrCreate(session, "/testRefObject");
```

Comment:
Instead of a hardcoded String `/testRefObject`, please create the resource name with:
` final String pid = UUID.randomUUID().toString();`